src/FsToolkit.ErrorHandling.TaskResult/TaskValidationCE.fs

```diff
@@ -0,0 +1,191 @@
+﻿namespace FsToolkit.ErrorHandling
+
+open System
+open System.Threading.Tasks
+
+[<AutoOpen>]
+module TaskValidationCE =
+
+    type TaskValidationBuilder() =
```

Comment:
Seems to be missing the [ResumableStateMachine](https://github.com/fsharp/fslang-design/blob/main/FSharp-6.0/FS-1087-resumable-code.md). See how `TaskResult` is implemented.

Reply:
I have never dealt with this before, I'll read about it and try to implement it.